api/openapi/specs/api.yaml

```
@@ -119,6 +119,186 @@ paths:
           $ref: '#/components/responses/GetDataplaneXDSConfigResponse'
         '400':
           $ref: './common/error_schema.yaml#/components/responses/BadRequest'
+        '500':
+          $ref: '#/components/responses/Internal'
+  /meshes/{mesh}/dataplanes/{name}/_layout:
+    get:
+      operationId: get-dataplanes-layout
+      summary: Get networking layout for this dataplane
+      description: Returns detailed dataplane networking layout with most important information on dataplane and list of its inbounds and outbounds
+      parameters:
+        - in: path
+          name: mesh
+          required: true
+          description: The mesh of the DPP to get the layout for.
+          schema:
+            type: string
+        - in: path
+          name: name
+          required: true
+          description: The name of the DPP within the mesh to get the layout for.
+          schema:
+            type: string
+      responses:
+        '200':
+          $ref: '#/components/responses/DataplaneNetworkingLayoutResponse'
+        '400':
+          $ref: './common/error_schema.yaml#/components/responses/BadRequest'
+        '500':
+          $ref: './common/error_schema.yaml#/components/responses/Internal'
+  /meshes/{mesh}/dataplanes/{name}/_policies:
+    get:
+      operationId: get-proxy-policy-conf
+      summary: Get policies for this dataplane
```

```suggestion
      summary: Get proxy policies for this dataplane
```
so it's clear this endpoint doesn't return all policies